### DISCUSSION TITLE
Custom fields per save, not per map?

### DISCUSSION THREAD

**Discussion by TuplexGaming** [Courseplay/Courseplay_FS22#1132] [Q&A] [answered]
I currently have 2 active save games on No Man's Land.  I've found that the custom fields from one save will appear on the other one.  Is there a way to separate them by save, rather than by map?  I'm on PC, and am willing to do some light editing to make this work.  I thought of making a copy of the map and changing the name to No Man's Land 2, for example, but I'm not sure how to get that to work with my existing save.

**Comment by Tensuko** [comment, marked as answer]
This is not per savegame, it is an extra folder for some reasons.
